contracts/plugins/assets/angle/USDAFiatCollateral.sol

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: BlueOak-1.0.0
+pragma solidity 0.8.19;
+
+import { CollateralConfig } from "../AppreciatingFiatCollateral.sol";
+import { ERC4626FiatCollateral } from "../ERC4626FiatCollateral.sol";
+
+/**
+ * @title USDA Fiat Collateral
+ * @notice Collateral plugin for USDA (Angle)
+ * tok = stUSD (ERC4626 vault)
+ * ref = USDA
+ * tar = USD
+ * UoA = USD
+ */
+
+contract USDAFiatCollateral is ERC4626FiatCollateral {
```

Comment:
`stUSDACollateral` or `StakedUSDACollateral` seem like more descript names, since the collateral is stUSDA and not USDA